frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -186,6 +191,15 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
               </FormHelperText>
             )}
           </FormGroup>
+          <StorageClassDropdown
+            onChange={handleStorageClass}
+            id="storageclass-dropdown"
+            data-test="storageclass-dropdown"
+            describedBy="storageclass-dropdown-help"
+            required={false}
+            name="storageClass"
+            filter={(p) => p?.provisioner === scResource?.provisioner}
```

Comment:
is cross clone between encrypted to non-encrypted SC allowed? In epic it is marked as out of scope

Reply:
we need to probably document ODF specific things... this is generic OCP UI, so need to have consistency between all StorageClasses...

need to decide what all things we should filter and what we leave for documentation... probably we can filter based on same provisioner (as we do now) + encrypted/non-encrypted as well...